maven-plugin/src/main/java/sortpom/VerifyMojo.java

```diff
@@ -93,6 +93,12 @@
     private String lineSeparator;
 
     /**
+     * Ignore line separators when comparing current POM with sorted one
+     */
+    @Parameter(property = "sort.ignoreLineSeparators", defaultValue = "true")
```

Comment:
Was here just because I modified the PluginParametersBuilder.setFormatting method. Moving the parameter out of that method and this change goes away too